.github/workflows/create-beta-pull-request.yml

```diff
@@ -38,7 +38,6 @@ jobs:
         GITHUB_TOKEN: ${{ secrets.PERSONAL_TOKEN }}
         MESSAGE_TITLE: Generated beta models and request builders using Typewriter
         MESSAGE_BODY: "This pull request was automatically created by the GitHub Action, **${{github.workflow}}**. \n\n The commit hash is _${{github.sha}}_. \n\n **Important** Check for unexpected deletions or changes in this PR."
-        REVIEWERS: peombwa,ddyett,zengin,nikithauc,baywet
         ASSIGNEDTO: ramsessanchez
```

Comment:
Should this be removed as well?

Reply:
it was, I think you're looking at an outdated commit.